wherescape/connectors/hubspot/send_data/send_company_object.py

```diff
@@ -0,0 +1,42 @@
+import logging
+
+from hubspot.crm.companies import (
+    SimplePublicObjectInput,
+    BatchInputSimplePublicObjectBatchInput,
+    ApiException,
+)
+
+"""
+Module where data is send to the companies object in Hubspot
+required scopes: crm.objects.companies.write
+"""
+
+
+def patch_company_on_id(id, properties, client):
```

Comment:
I think for all the functions in send_date we can make it to one, but we add a string to the functions that indicates if a company, contact or deal is being send! This will make it a lot easier if you need to change something in the send function, because you don't need to update every copy of the function!